src/components/Calculator.js

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import './Calculator.css';
+import './calculator.css';
```

Comment:
* Kindly fix this error I get as you are importing `calculator.css` instead of `Calculator.css` as the file is in upper case 
![image](https://user-images.githubusercontent.com/52497246/212493527-dd5da27f-cb93-4ef4-a6ab-20a66edfab2a.png)